docs/old-wiki/usr.adding.new.package.custom.scheme.md

```diff
@@ -164,4 +164,4 @@ Note that such CMake variables like:
 * [CMAKE_CONFIGURATION_TYPES](http://www.cmake.org/cmake/help/v3.2/variable/CMAKE_CONFIGURATION_TYPES.html)
 * [BUILD_SHARED_LIBS](http://www.cmake.org/cmake/help/v3.2/variable/BUILD_SHARED_LIBS.html)
 
-must to be checked manually for each custom build scheme (see [CMake package default behaviour](https://github.com/ruslo/hunter/wiki/usr.adding.new.package#default-behaviour))
\ No newline at end of file
+must to be checked manually for each custom build scheme (see [CMake package default behaviour](https://github.com/ruslo/hunter/wiki/usr.adding.new.package#default-behaviour))
```

Comment:
what's the change of this line, I don't see a difference in GitHub view :thinking:

Reply:
My editor has inserted a newline at the end